tests/integration/openapi3-express.spec.ts

```diff
@@ -60,11 +62,6 @@ describe('OpenAPI3 Express Server', () => {
       word: 'word',
       fourtyTwo: 42,
       intersectionAlias: { value1: 'value1', value2: 'value2' },
-      intersectionAlias2: { value1: 'value1', value2: 'value2', value3: 'string' },
```

Comment:
What's the rationale for removing these?

Reply:
Was not intended. I have added a commit fix unintended changes in this file.